scripts/build-env.js

```diff
@@ -1,8 +1,44 @@
 const execSync = require('child_process').execSync
 const lernaJson = require('../lerna.json')
 
+/**
+ * Allows to produce slightly custom builds of bundles and packages to be used in various
+ * environment. On top of allowing some internal use cases, it also drives the SDK version being
+ * used when requesting the intake (see below).
```

Comment:
```suggestion
 * environment, it allows:
 * - some internal uses cases
 * - different strategies for sdk_version sent to the intake (see below)
```

I had a hard time understanding what you referred to with SDK version